Application/EdFi.Ods.Api/Security/Authorization/Repositories/AggregateRootCriteriaProviderDecoratorBase.cs

```diff
@@ -56,77 +56,101 @@ public ICriteria GetCriteriaQuery(TEntity specification, IQueryParameters queryP
 
             var authorizationFilters = _authorizationFilterContextProvider.GetFilterContext();
```

Comment:
This file needs to be reviewed for the true intent of changes because I see old code, unrelated to the record ownership work, getting reintroduced here! What else is getting reintroduced that we haven't detected?

See my comment below. The lineage looks messed up, and it is alarming for the whole PR, but perhaps the trouble is isolated to this class.

Reply:
I noticed that you have reverted old changes and sync in main